### PR TITLE
Fix/e2e 852

### PR DESCRIPTION
**Pull request by 6l20**
## PR Description

- Re organise the k8s chart with everything but orchestrate and Vault remote
- Align all environments

## Fixed Issue(s)

Partly contributing to orchestrate/#852

## Documentation

- [ ] I thought about documentation and added the `documentation` label to this PR if updates are required.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## v7.0.0 (Unreleased)
+
+ * Remove everything not orchestrate and Vault
```
Comment (ggarri) [inline CHANGELOG.md:7]: This comment is hard to understand, could you improve it?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
...
+ * QKM deployment now relies on quorum-key-manager chart only
+ * Everything infra is moved to orchestrate-infra
```
Comment (ggarri) [inline CHANGELOG.md:9]: COuld you specify which are the infra components moved?

File: environments/common.yaml.gotmpl [hunk 3]
```diff
@@ -64,114 +97,12 @@ vault:
...
-
 qkm:
-  enabled: true
-  url: "http://quorum-key-manager-quorumkeymanager:8080"
-  proto: https
-  fullname: "quorumkeymanager"
   orchestrate:
     storeName: "eth-accounts"
-    apiKey: {{ env "QKM_API_KEY" | default "NmwyMCB3YXMgaGVyZQ==" }}
+    apiKey: {{ env "QKM_API_KEY" | default "YWRtaW4tdXNlcg==" }}
```
Comment (ggarri) [inline environments/common.yaml.gotmpl:103]: I would rather not have any default value

File: environments/staging.yaml [hunk 4]
```diff
@@ -9,33 +9,8 @@ orchestrate:
...
-    slaveCount: 2
-
 vault:
   replicaCount: 3
 
 qkm:
-  enabled: true
-  replicaCount: 3
\ No newline at end of file
+  url: https://qa-qkm.ops.consensys.net
```
Comment (ggarri) [inline environments/staging.yaml:16]: typo: `https://staging-qkm.ops.consensys.net`

**Review by ggarri — APPROVED** [review]

File: environments/common.yaml.gotmpl (hunk 3 above)
Comment (6l20) [reply environments/common.yaml.gotmpl:103]: This is the default QKM expected value, so it gives the user a good hint according to me. For that reason I would rather keep it

File: environments/staging.yaml (hunk 4 above)
Comment (6l20) [reply environments/staging.yaml:16]: Right

**Pull request merged**